clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -454,6 +455,59 @@ public boolean matches(ClientRequest request) {
     }
 
     @Test
+    public void testCommitsFetchedDuringAssign() {
+        String topic = "topic";
+        final TopicPartition partition1 = new TopicPartition(topic, 0);
+        final TopicPartition partition2 = new TopicPartition(topic, 1);
+
+        long offset1 = 10000;
+        long offset2 = 20000;
+
+
```

Comment:
In fact, I think I'd remove all empty lines all the way to `autoCommitIntervalMs` as I don't think they add much.